src/esn.inbox.libs/app/services/mailboxes/mailboxes-service.js

```diff
@@ -252,11 +265,15 @@ angular.module('esn.inbox.libs')
       }
 
       return withJmapClient(function(jmapClient) {
-        return jmapClient.getMailboxes()
+        return jmapClient
+          .mailbox_get({
+            accountId: Object.keys(jmapClient.getSession().accounts)[0],
```

Comment:
This is problematic. The caller to mailbox_get should not need to have knowledge of internal data structures.
You have to add getter functions

Reply:
Added getter functions, and getSession is not used anymore (but still present in the library if needed).